addons/hr_holidays/models/hr_employee_base.py

```diff
@@ -281,3 +281,6 @@ def write(self, values):
             ])
             allocations.write(hr_vals)
         return res
+
+    def _get_timezone(self):
+        return self.resource_calendar_id.tz or self.tz or self.company_id.resource_calendar_id.tz
```

Comment:
you have _get_tz in employee (discovered yesterday :joy: ) maybe you need to modify the function to have the right order :thinking:

Reply:
i think :grimacing:

Reply:
Oh nice !